src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServicePermissionTests.cs

```diff
@@ -55,7 +55,7 @@ await PermissionTestUtils.PolicyCheckBuilder<SecurityPolicies>()
         public async Task ListPublicationSummaries()
         {
             await PermissionTestUtils.PolicyCheckBuilder<SecurityPolicies>()
-                .ExpectCheckToFail(SecurityPolicies.CanManageUsersOnSystem)
+                .ExpectCheckToFail(SecurityPolicies.IsBauUser)
```

Comment:
Would probably check the policy `CanViewAllPublications` instead, which amounts to the same really, but until we replace all BAU-only SecurityPolicies with IsBauUser, might as well use it here?

Reply:
Ah yep, I didn't realise that one existed, thanks. I've switched it to use `CanViewAllPublications` now, thanks.